packages/executor/src/utils/constants.ts

```diff
@@ -12,13 +12,14 @@ export const customChains = [
       browserURL: 'https://goerli-optimism.etherscan.io',
     },
   },
+  {
+    network: 'arbitrum-goerli',
+    chainId: 421613,
+    urls: {
+      apiURL: 'https://api-goerli.arbiscan.io/api',
+      browserURL: 'https://goerli.arbiscan.io/',
+    },
+  },
```

Comment:
I tested this on Arbitrum, b/c live network deployments still were not working for me.